tensorflow/python/kernel_tests/nn_ops/pooling_ops_3d_test.py

```diff
@@ -48,6 +48,14 @@ def GetTestConfigs():
 @test_util.with_eager_op_as_function
 class PoolingTest(test.TestCase):
 
+  def _DtypesToTest(self, use_gpu):
+    if use_gpu:
+      return [dtypes.float32]
+    else:
+      # It is important that float32 comes before float16 here, as we will be
+      # using its gradients as reference for bf16 gradients.
```

Comment:
This PR doesn't seem to use fp32 values to check bf16 values anymore. Should we remove this comment / order enforcement?